locales/es/LC_MESSAGES/hidden-curriculum.po

```diff
@@ -132,561 +168,665 @@ msgid ""
-"is a freelancer. He is a native French speaker with basic English "
-"knowledge. He has completed some undergraduate courses at the university "
+"is a freelancer. He is a native French speaker with basic English knowledge. "
+"He has completed some undergraduate courses at the university "
 "and has engineering background. He leads a citizen-driven grassroots "
 "organization that performs investigative journalism, reporting on "
 "governmental policies around ecological issues like environmental "
-"sustainability and climate change, and social issues like femicides. "
+"sustainability, climate change, and social issues like femicides. "
 "Mohamed’s goals are to find a way to have the grassroots organisation "
 "become sustainable, including paying wages to staff and supporting "
 "infrastructure like cloud computing and computers. He would also like to "
 "receive travel grants to support additional investigative journalism. "
 "Mohamed has never received any grant before (neither local or "
 "international). Mohamed doesn’t know where to find grants and lacks "
 "familiarity with the administrative process involved; moreover, he has no"
-" experience in designing large-scale research projects. Some of the "
+"experience in designing large-scale research projects. Some of the "
 "questions Mohamed has are: What are the requirements of a donor? How do I"
-" report to them? How can I use the money? How do I pay people?"
-msgstr ""
+"report to them? How can I use the money? How do I pay people?"
+msgstr ""
+"Mohamed tiene veintitantos años. Es de Mali, donde vive"
+"con sus padres y varios hermanos. Tiene ingresos esporádicos ad hoc y "
+"Es un profesional independiente. Es un hablante nativo de francés con conocimientos básicos de inglés. "
+"Ha completado algunos cursos de pregrado en la universidad"
+"y tiene experiencia en ingeniería. Dirige una organización de base impulsada por los ciudadanos "
+"organización que realiza periodismo de investigación, informando sobre"
+"políticas gubernamentales en torno a cuestiones ecológicas como el medio ambiente"
+"sostenibilidad, cambio climático y cuestiones sociales como los feminicidios".
+"El objetivo de Mohamed es encontrar la manera de tener organización de base"
+"volverse sostenible, incluido el pago de salarios al personal y el apoyo "
+"infraestructura como computación en la nube y computadoras. También le gustaría "
+"recibir becas de viaje para apoyar el periodismo de investigación adicional".
+"Mohamed nunca antes había recibido ninguna subvención (ni local ni "
+"internacional). Mohamed no sabe dónde encontrar subvenciones y le falta "
+"familiaridad con el proceso administrativo involucrado; además, no tiene"
+"experiencia en el diseño de proyectos de investigación a gran escala. Algunos de los "
+"Las preguntas que tiene Mohamed son: ¿Cuáles son los requisitos de un donante? ¿Cómo lo hago?"
+"¿Informarles? ¿Cómo puedo usar el dinero? ¿Cómo le pago a la gente?"
```

Comment:
```suggestion
"Mohamed tiene veintitantos años. Es de Mali, donde vive "
"con sus padres y varios hermanos. Tiene ingresos esporádicos ad hoc y "
"es autónomo. Es un hablante nativo de francés con conocimientos básicos de inglés. "
"Ha completado algunos cursos de grado en la universidad "
"y tiene experiencia en ingeniería. Dirige una organización ciudadana comunitaria "
"que realiza periodismo de investigación, informando sobre "
"políticas gubernamentales en cuestiones ecológicas como "
"la sostenibilidad y el cambio climático, y cuestiones sociales como los feminicidios. "
"El objetivo de Mohamed es encontrar la manera de hacer su organización "
"sostenible, incluyendo el pago de salarios al personal y la infraestructura "
"de soporte como computación en la nube y computadoras. También le gustaría "
"recibir becas de viaje para apoyar investigaciones periodísticas adicionales. "
"Mohamed nunca ha recibido ninguna subvención (ni local ni "
"internacional). Mohamed no sabe dónde encontrar subvenciones y le falta "
"familiaridad con el proceso administrativo involucrado; además, no tiene"
"experiencia en el diseño de proyectos de investigación a gran escala. Algunos de las "
"las preguntas que se hace Mohamed son: ¿Cuáles son los requisitos de un donante? ¿Cómo los informo?"
"¿Cómo puedo usar el dinero? ¿Cómo pago a la gente?"
```

not 100% sure if "engineering background" means he has worked as an engineer (would be the current "tiene experiencia en ingeniería") or if he has received some formation to be work as an engineer (would be "tiene formación en ingeniería" instead).